cmd/swarmctl/service/flagparser/flags.go

```diff
@@ -32,6 +32,7 @@ func AddServiceFlags(flags *pflag.FlagSet) {
 
 	flags.Uint64("update-parallelism", 0, "task update parallelism (0 = all at once)")
 	flags.String("update-delay", "0s", "delay between task updates (0s = none)")
+	flags.String("update-on-failure", "pause", "action on failure during update (pause|continue)")
```

Comment:
It's part of of the service spec, so it's fine to set it on create. The setting applies for future updates unless you override it.

I'd like the flag to start with `update` to group it with the other update flags.